automatic/nodejs.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$filePath32 = "$toolsPath\node-v7.4.0-x86.msi"
+$filePath64 = "$toolsPath\node-v7.4.0-x64.msi"
+$installFile = if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
+                      Write-Host "Installing 64 bit version"; $filePath64 }
+               else { Write-Host "Installing 32 bit version"; $filePath32 }
+
+$packageArgs = @{
+    PackageName    = 'nodejs.install'
+    FileType       = 'msi'
+    SoftwareName   = 'Node.js'
+    File           = $installFile
+    SilentArgs     = "/quiet"
+    ValidExitCodes = @(0)
+}
+Install-ChocolateyInstallPackage @packageArgs
+
+$nodePath = "$env:SystemDrive\Program Files\nodejs"
```

Comment:
please don't use hard coded paths, look into using the [Get-AppInstallLocation](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/extensions/chocolatey-core.extension/extensions/Get-AppInstallLocation.ps1) located in the chocolatey-core.extension package instead.